.github/ISSUE_TEMPLATE/implementing-new-platform-light.md

```diff
@@ -0,0 +1,38 @@
+# Implementing a new supported platform (light)
```

Comment:
From discussions during the community meeting, one thing we said was that these platforms would be supported at a different level. Should we reflect that in this section? I think there was agreement on the term "emerging" which could replace the "light" term here.